examples/platform/policy_data_collection.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python
+# *******************************************************
+# Copyright (c) VMware, Inc. 2020-2023. All Rights Reserved.
+# SPDX-License-Identifier: MIT
+# *******************************************************
+# *
+# * DISCLAIMER. THIS PROGRAM IS PROVIDED TO YOU "AS IS" WITHOUT
+# * WARRANTIES OR CONDITIONS OF ANY KIND, WHETHER ORAL OR WRITTEN,
+# * EXPRESS OR IMPLIED. THE AUTHOR SPECIFICALLY DISCLAIMS ANY IMPLIED
+# * WARRANTIES OR CONDITIONS OF MERCHANTABILITY, SATISFACTORY QUALITY,
+# * NON-INFRINGEMENT AND FITNESS FOR A PARTICULAR PURPOSE.
+
+"""Example script which manipulates rules in the Data Collection component of policy.
+
+There are three methods that demonstrate interactions with the Data Collection Policy.
+The first prints summary information about each policy, and iterates through each data_collection rule
+This is an example of the command line to execute.
+> python examples/platform/policy_data_collection.py --profile EXAMPLE_CREDENTIALS list_policies
+
+The second method enables Auth Event collection (a type of data collection rule) on an existing policy.
+Command line prompts ask for required info and defaults are provided.
+This is an example of the command line to execute. The script will prompt on the command line for Policy Id.
+> python examples/platform/policy_data_collection.py --profile EXAMPLE_CREDENTIALS enable_auth_event_rule
+
+To disable an Auth Event policy,
+> python examples/platform/policy_data_collection.py --profile EXAMPLE_CREDENTIALS disable_auth_event_rule
+
+The API specification for underlying APIs is available on the Developer Network:
+https://developer.carbonblack.com/reference/carbon-black-cloud/platform/latest/policy-service/#rule-config---data-collection
+"""
+
+import sys
+from cbc_sdk.helpers import build_cli_parser, get_cb_cloud_object
+from cbc_sdk.platform import Policy
+
+
+def list_policy_summaries(cb, args):
```

Comment:
I am not seeing args being used in this function.

Reply:
removed.